projects/metis/src/app/dataset/workflow/workflow.component.ts

```diff
@@ -385,10 +387,10 @@ export class WorkflowComponent extends SubscriptionManager implements OnInit {
         this.workflowForm.controls.harvestUrl.setValue(thisWorkflow.url.trim().split('?')[0]);
         this.workflowForm.controls.setSpec.setValue(thisWorkflow.setSpec);
         this.workflowForm.controls.metadataFormat.setValue(thisWorkflow.metadataFormat);
-        this.workflowForm.controls.incrementalHarvest.setValue(thisWorkflow.incrementalHarvest);
+        //this.workflowForm.controls.incrementalHarvest.setValue(thisWorkflow.incrementalHarvest);
       }
     }
-    this.enableIncrementalHarvestingFieldIfAvailable(workflow);
+    //this.enableIncrementalHarvestingFieldIfAvailable(workflow);
   }
 
   /** extractPluginParamsExtra
```

Comment:
I think in this file we also need to remove the reference to `ParameterFieldName.incrementalHarvest` in line 499 (method `formatFormValues()`), just to be safe. Does that make sense?

Reply:
Done!